tools/baml-bench/ui/app/globals.css

```
@@ -0,0 +1,738 @@
+
+details.turn-block pre,
+details.turn-tool pre {
+  font-family: ui-monospace, "SF Mono", Menlo, monospace;
+  font-size: 12px;
+  line-height: 1.55;
+  color: var(--fg);
+  background: var(--bg);
+  padding: 8px 10px;
+  margin: 4px 0 0 18px;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)